abdulfataiaka/pluralsight-admin
# #161352701 Add courses count on the header

#### What does this PR do
- Adds courses count on the header

#### Description of work done
- Add a styled circle beside the courses link on the header

#### How should it be manually tested
- Clone the repository `git clone https://github.com/abdulfataiaka/pluralsight-admin.git`
- Checkout to this branch
- Ensure you see courses count on the header

#### Relevant pivotal tracker story
[161352701](https://www.pivotaltracker.com/story/show/161352701)

#### Screenshots
<img width="1440" alt="screen shot 2018-10-19 at 7 40 24 pm" src="https://user-images.githubusercontent.com/32123313/47237399-d6806180-d3d6-11e8-9dd1-ede26a9b4d10.png">
